pip-requirements.txt

```diff
@@ -46,6 +46,7 @@ poetry
 tox
 pre-commit
 ipython>=7.29.0
+jupyterlab
```

Comment:
jupyterlab was previously only tied into the constraints because of the otherwise unused `docker/prognostic_run/requirements/notebook.txt` file.